src/Microsoft.PowerApps.TestEngine/TestEngine.cs

```diff
@@ -123,6 +123,19 @@ public async Task<string> RunTestAsync(FileInfo testConfigFile, string environme
             {
                 if (TestLoggerProvider.TestLoggers.ContainsKey(testRunId))
                 {
+                    if (!string.IsNullOrEmpty(testRunDirectory))
```

Comment:
I feel like this could create a chicken and egg problem i.e. if there is an issue with command line arguments the user has entered, then those are errors BEFORE the test execution even began, and hence the directories are not setup yet by design. I feel we should divide the error scopes to "before test execution" and "from test execution".

For instance, if I enter a bad path for a test plan file, I would want these errors to be dumped at the `log` folder in the root directory that indicates runtime errors before a test was even started.

All other errors from the point where the test plan was read can go into the TestOutput\testId\ folder.